portfolio/src/main/webapp/scripts/controllers/request_auth.js

```diff
@@ -0,0 +1,34 @@
+import { displayError } from '../views/forms/utils.js';
+
+const handleError = (error) => {
```

Comment:
This method shouldn't be necessary, just do displayError(error.message) in your catch block.